src/pages/documentation/adobe-analytics/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## March 27, 2023
+
+### Android Analytics 2.0.2
+
+* Fixed a bug that was causing status Unavailable to be shown in Adobe Analytics view in Assurance for some track events sent soon after app launch.
```

Comment:
Just to confirm, is the status' name "Unavailable"?

If so, I'd go with "... that was causing the "Unavailable" status to be shown..." just to be a bit clearer.